### ISSUE TITLE
Focus action pane by default in some cases

### ISSUE THREAD

**Issue by bluss** [kupferlauncher/kupfer#57]
Let's consider focusing the action pane in a number of cases (instead of the primary object pane; the action pane is where actions appear, to the right of the primary object pane by default).

- [ ] When an action returns a result
  - *Copy To..* (*File Actions* plugin)
  - *Select in Kupfer* (*Higher-order Actions* plugin)
- [ ] When files are posted
  - `kupfer FILE ...` from terminal or via dbus
  - Drop files on the kupfer starter icon
- [ ] When text is posted
  - Currently this lets you edit the text. I don't think I want to change this

Help Wanted Tag: because user input is wanted.

Tangential issues: Accelerator key to select each pane (primary object, action, indirect object)? Alt + 1, 2, 3 would be natural.

**Comment by khurshid-alam** [comment]
Can we use Right-Arrow-Key to switch to Action pane? I have to pres Tab (and then the down-arrow to list through all other actions) to switch?

**Comment by bluss** [comment]
I don't think so, right arrow is already used to descend into things (into collections and directories). New in last version was at least that finally one can add a key shortcut to go to pane 1, 2, and 3, So you can use ctrl  + 2 for example to switch to actions. Action accelerators PR #58 will probably help with some of the action switching pain too.

**Comment by khurshid-alam** [comment]
Ok. Let's carry on discussion on #58 page

**Comment by bluss** [comment]
Lol, I found a shortcut in the code I didn't know: Shift + Down is the same as Tab to switch to action pane, then Down. That's kind of what was asked for. (I'm not entirely sure when this was added; it was maybe part of “type with shift to search for actions” which was an unused experiment.)
